features/admin.claims.v1/components/edit/local-claim/edit-basic-details-local-claims.tsx

```diff
@@ -953,6 +957,18 @@ export const EditBasicDetailsLocalClaims: FunctionComponent<EditBasicDetailsLoca
                         hint={ t("claims:local.forms.descriptionHint") }
                         readOnly={ isSubOrganization() || isReadOnly }
                     />
+
+                    <Field.Checkbox
+                        ariaLabel="multiValued"
```

Comment:
ariaLabel needs to be human friendly text.

```suggestion
                        ariaLabel={ t("claims:local.forms.multiValued.label") }
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7968/commits/28773da69fdca6ec31f2cfb3529c2b4a79d875eb